content/guides/models/registry/create_registry.md

```diff
@@ -7,23 +7,71 @@ title: Create a custom registry
 weight: 2
 ---
 
-Create a [custom registry]({{< relref "./registry_types.md#custom-registry" >}}) for each step of your ML workflow.
+A custom registry offers flexibility and control over the artifact types that you can use, the visibility of the registry to other teams, and more.
```

Comment:
Registry was previously capitalized most of the time
```suggestion
A custom registry offers flexibility and control over the artifact types that you can use, allows you to restrict the registry's visibility, and more.
```

Reply:
Ah. It should only be capitalized if its a proper noun... 

I  would "App" to try to help distinguish proper vs common noun usage of "registry".  But since removed it...

E.g. "Registry App" vs "a registry ..."